README.md

```diff
@@ -27,12 +28,13 @@ following entitlements are supported:
 - [FIPS 140-2 Certified Modules (and optional non-certified patches)](https://ubuntu.com/security/fips)
 - [Livepatch Service](https://ubuntu.com/security/livepatch)
 
-## Quickstart
 
-`ua` is already installed on every Ubuntu system. Try it out by running `ua help`!
+If you need any of those services in your machine, UA is the right tool for you.
+Furthermore, UA is already installed on every Ubuntu system. Try it out by running `ua help`!
 
-To learn the basic commands and enable professional Ubuntu services in a sandbox environment, follow our [Getting Started Tutorial](./docs/tutorials/basic_ua_commands.md).
+## Quickstart
 
+To learn the basic commands and enable professional Ubuntu services in a sandbox environment, follow our [Getting Started Tutorial](./docs/tutorials/basic_ua_commands.md).
```

Comment:
I added this, but now thinking it doesn't add much value over the first entry in the tutorials list. We can just remove it
```suggestion
```